apps/sso/components/login-card/login-card.tsx

```diff
@@ -137,3 +137,22 @@ const StyledErrorText = styled(Text)`
   padding-top: 1rem;
   color: red;
 `;
+const Input = styled.input`
+  border: solid 0.1rem #bcbcbc;
+  background-color: #f8f8f8;
+  border-radius: 0.3rem;
+  padding: 10px;
+  padding-left: 15px;
+  font-family: InterVariable, sans-serif;
+  font-size: 1.1rem;
+  color: #676767;
```

Comment:
nit: For color hexcodes like this, we usually recommend having them as constants either on top of the file or in another file so that there is consistency between pages.

Reply:
Also I think the color should be black, or at least a darker color (current gray is too light)